source-code/ide-extension/src/main.ts

```diff
@@ -55,33 +57,39 @@ async function main(args: { context: vscode.ExtensionContext }): Promise<void> {
     options: potentialConfigFileUris.map((uri) => uri.path),
     to: activeTextEditor.document.uri.path,
   });
-  const configModule: InlangConfig = (await import(closestConfigPath)).config;
+
+  // Change current working directory to configuration file directory, so relative paths to the languages files work.
+  // Otherwise the current working directory path is the directory where the node binary resides.
+  process.chdir(dirname(closestConfigPath));
```

Comment:
Deprecates `configPath`.

Reply:
Have you verified whether VSCode isolates extensions? Otherwise, we might be crashing other extensions with this line of code. 

Furthermore, I remember running into this problem. We need to develop our own filesystem implementation that does not rely on node internals with pluggable backends e.g. virtual filesystem, local storage filesystem, etc. That's part of https://github.com/inlang/inlang/issues/278

Reply:
After some thinking, I think it is still the best solution, if it works in all environments of the ide-extension. My reasons are:
-  The core shouldn't need to handle this, as the to be imported URI is provided from the outside and the outside should make sure that the URI work in its environment.
- When we start handling paths inside the core, we open a little Pandoras box. We would need to load `node:path` and also start to use one of the path polyfills for enabling the browser. 

If the currently implementation does it's job, we should continue work with it, otherwise I suggest a callback function on the `environment` passed to `$import`, which resolves given URIs.

Furthermore I've opened a discussion about this here: https://github.com/microsoft/vscode-discussions/discussions/466

I suggest to move onward with this solution for now.

Reply:
Yes, let's move on. 

If we are not breaking other extensions, we should be fine. 

> The core shouldn't need to handle this, as the to be imported URI is provided from the outside and the outside should make sure that the URI work in its environment.

Agree. The filesystem implementation we are using should handle the working directory.